src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -44,6 +62,12 @@ public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator
         Person[] persons = getPersons();
 
         // TODO использовать Arrays.sort
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                return o1.getLastName().equals(o2.getLastName()) ? o1.getFirstName().compareTo(o2.getFirstName()) : o1.getLastName().compareTo(o2.getFirstName());
```

Comment:
Did you compare last name and first name on purpose after ":"?

Reply:
Thank you!